packages/web/src/common/services/audius-backend/AudiusBackend.ts

```diff
@@ -541,16 +546,22 @@ export const audiusBackend = ({
     secondaries: string[],
     reason: string
   ) {
-    recordAnalytics(Name.CREATOR_NODE_SELECTION, {
-      endpoint: primary,
-      selectedAs: 'primary',
-      reason
+    recordAnalytics({
+      eventName: Name.CREATOR_NODE_SELECTION,
```

Comment:
wow epic job changing all these

Reply:
Thanks! idk if this is the final interface we should use but updating web was the quickest route